style.css

```diff
@@ -0,0 +1,857 @@
+}
+
+.article-continue-img p {
+  font-size: 1.4rem;
+  color: var(--color-text-img);
+  margin: 1rem 0;
+  width: 100%;
+}
+
+.small-text p {
+  font-size: 1.2rem;
+  color: var(--color-text-img);
+  margin: 2rem auto;
+  width: 56%;
+}
+
+.small-text a {
+  color: var(--color-text-img);
+  position: relative;
+  margin-right: 2rem;
+  transition: all 0.4s;
+}
+
+.small-text a:not(:last-child)::before {
+  content: "";
+  width: 0.1rem;
+  height: 2rem;
+  position: absolute;
+  top: 0;
+  right: -1.5rem;
+  background-color: var(--color-text-img);
+}
+
+.article-continue-bottom-box {
+  justify-content: space-between;
+  width: 56%;
+  margin: 0 auto;
+}
+
+.continue-left button {
+  width: 20rem;
+  margin-left: -0.2rem;
+}
+
+.margin-circle-top {
+  margin-top: 1.5rem;
+  margin-right: 1rem;
+}
+
+.article-title {
+  width: 92%;
+  margin: 0 auto;
+}
+
+.article-title-2 {
+  width: 56%;
+  margin: 2rem auto !important;
+}
+
+.article-link {
+  text-decoration: underline;
+  color: var(--color-navbar-button);
+  transition: all 0.4s;
+  font-weight: 600;
+}
+
+/* -------------------------- Recomendations Section ----------------------------- */
+.recomendations-box {
+  display: grid;
+  grid-template-columns: minmax(max-content, 1fr) 31.5rem;
+  grid-template-rows: min-content;
+  width: 50%;
+  margin: 0 auto;
+  justify-items: center;
+  justify-content: center !important;
+  margin-top: 6rem;
+}
+
+.recomendations {
+  display: grid;
+  grid-template-columns: minmax(max-content, 1fr);
+  grid-template-rows: repeat(2, min-content);
+}
+
+.recomendations-top {
+  grid-column: 1 / 4;
+  grid-row: 1 / 2;
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: repeat(2, min-content);
+  grid-gap: 2rem;
+  width: 80%;
+  position: relative;
+}
+
+.items h4 {
+  font-size: 1.8rem;
+  font-weight: 400;
+}
+
+.items p {
+  font-size: 1.4rem;
+  color: var(--color-add-span);
+}
+
+.recomendations-bottom {
+  grid-column: 1 / 4;
+  grid-row: 2 / 3;
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: min-content;
+  grid-gap: 2rem;
+  width: 80%;
+  position: relative;
+  margin-top: 8rem;
+  margin-bottom: 6rem;
+}
+
+.most-popular {
+  grid-column: 2 / 3;
+  grid-row: 1 / 2;
+  position: relative;
+  margin-left: 10rem;
+}
+
+.recomendations-top::before {
+  content: "More in Space and Astronomy";
+  width: 125%;
+  height: 0.1rem;
+  background-color: black;
+  position: absolute;
+  top: -4rem;
+  left: 0;
+  font-size: 1.6rem;
+  font-weight: 700;
+}
+
+.recomendations-bottom::before {
+  content: "Editors’ Picks";
+  width: 125%;
+  height: 0.1rem;
+  background-color: black;
+  position: absolute;
+  top: -4rem;
+  left: 0;
+  font-size: 1.6rem;
+  font-weight: 700;
+}
+
+.most-popular::before {
+  content: "Most Popular";
+  width: 130%;
+  height: 0.1rem;
+  position: absolute;
+  top: -4rem;
+  left: -10rem;
+  background-color: black;
+  font-size: 1.6rem;
+  font-weight: 700;
+}
```

Comment:
The width of this line should be the same from the items below.
| Yours | Original |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/87702302-9c8cec80-c76f-11ea-93ef-9d2d05c80ac3.png) | ![image](https://user-images.githubusercontent.com/61479648/87702472-d6f68980-c76f-11ea-9bc0-4adc135fad9e.png) |

Reply:
Thank you for the review! We fixed those two details